packages/@aws-cdk/aws-servicecatalogappregistry/lib/application.ts

```diff
@@ -170,14 +170,16 @@ abstract class ApplicationBase extends cdk.Resource implements IApplication {
   }
 
   /**
-   * Associate all stacks present in construct's aspect with application.
+   * Associate all stacks present in construct's aspect with application, including cross-account stacks.
    *
    * NOTE: This method won't automatically register stacks under pipeline stages,
    * and requires association of each pipeline stage by calling this method with stage Construct.
    *
    */
   public associateAllStacksInScope(scope: Construct): void {
-    cdk.Aspects.of(scope).add(new StageStackAssociator(this));
+    cdk.Aspects.of(scope).add(new StageStackAssociator(this, {
+      enableCrossAccount: true,
```

Comment:
I don't mind this defaulting to `true`, but why does this need to be specified at all? Can we make `enableCrossAccount` `true` by default on the `StageStackAssociator`?

Reply:
Yes, I think defaulting to `true` makes sense.

Reply:
I have since discussed this with the team and we strongly favor making this `false` on `StageStackAssociator` because it is more likely that customers have not setup properly for cross-account cases. Setting it to `false` makes this safe assumption so that the default behavior doesn't break any customers.

Then, we'd have to set the value to `true` here as we want `associateAllStacksInScope()` to include cross-account stacks, and in this case the customer would be explicitly associating all stacks by calling this method.